docs/sphinx/using/simulators.rst

```diff
@@ -91,6 +91,45 @@ To execute a program on the :code:`nvidia-mgpu` target, use the following comman
 
   This backend requires an NVIDIA GPU, CUDA runtime libraries, as well as an MPI installation. If you do not have these dependencies installed, you may encounter an error stating `Invalid simulator requested`. See the section :ref:`dependencies-and-compatibility` for more information about how to install dependencies.
 
+.. note:: 
```

Comment:
While informative, I wonder if this belongs on the 'simulators' page?

Reply:
Agreed - it seems that we can completely delete this all by recommending the hpc sdk instead.